packages/core/test/foundry/data-verification-mechanism/fork-tests/CommonDataVerificationMechanismTest.sol

```diff
@@ -0,0 +1,53 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.0;
+
+import "../../fixtures/common/CommonTestBase.sol";
+
+import "../../../../contracts/data-verification-mechanism/implementation/VotingV2.sol";
+import "../../../../contracts/data-verification-mechanism/interfaces/VotingAncillaryInterface.sol";
+import "../../../../contracts/common/interfaces/ExpandedIERC20.sol";
+
+contract CommonDataVerificationMechanismTest is CommonTestBase {
+    VotingV2 voting;
+    ExpandedIERC20 votingToken;
+
+    bool shouldRunTest;
+
+    address registeredRequester;
+    address governor;
+
+    bytes32 identifier = bytes32("YES_OR_NO_QUERY");
+    bytes ancillaryData = bytes("Some data");
+    uint256 gatMeetingAmountOfTokens = 6e24;
+    uint256 requestTime = 420;
+
+    function _commonSetup() public {
+        uint256 chainId = block.chainid;
+        shouldRunTest = (chainId == 1 || chainId == 5);
+        if (!shouldRunTest) return; // Exit early if we are not executing forked tests.
```

Comment:
nit:

Is this meant to house _any_ DVM tests? Or just specifically forked ones?

I noticed that shouldRunTest is based on whether we're on a fork, but everything else in here seems general. Thoughts on renaming this variable to shouldRunForkTests?

Reply:
nope, this is meant to house only `fork-tests`, as it is located in this directory. agreed though that the variable name is not as explicit as it should be. I'll update it.